fix (curriculum): added functionality to merge steps 21 - 22 and steps 41 - 42 (#54917)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/646ddd3f9f97a0667b964bdb.md

---
id: 646ddd3f9f97a0667b964bdb
title: Step 40
challengeType: 0
dashedName: step-40
---

# --description--

> Create two `div` elements, one with the class `cat-left-inner-eye` inside the `.cat-left-eye` element and the second with the class `cat-right-inner-eye` inside the `.cat-right-eye` element.

Reviewer: ```suggestion
Create two `div` elements, one with the class `cat-left-inner-eye` inside the `.cat-left-eye` element and an other with the class `cat-right-inner-eye` inside the `.cat-right-eye` element.
```

# --hints--

You should not change the existing `div` elements with the class `cat-left-eye` and `.cat-right-eye`.

```js
assert(document.querySelectorAll('div.cat-left-eye').length === 1);
assert(document.querySelectorAll('div.cat-right-eye').length === 1);
```

> You should have two `div` elements inside your `.cat-left-eye` element and `.cat-right-eye` element.

Reviewer: please have two different tests here

```js
assert(document.querySelectorAll('.cat-left-eye div').length === 1);
assert(document.querySelectorAll('.cat-right-eye div').length === 1);
```

> Your `div` elements should have the class `cat-left-inner-eye` and `cat-right-inner-eye`.

Reviewer: please have two different tests here

assert(document.querySelectorAll('div.cat-left-eye').length === 1);
```

You should not change the existing `div` element with the class `.cat-right-eye`.

```js
assert(document.querySelectorAll('div.cat-right-eye').length === 1);
```

You should have a `div` element inside your `.cat-left-eye` element.

```js
assert(document.querySelectorAll('.cat-left-eye div').length === 1);
```

You should have a `div` element inside your `.cat-right-eye` element.

```js
assert(document.querySelectorAll('.cat-right-eye div').length === 1);
```

> Your `div` element should have the class `cat-left-inner-eye`.

Reviewer: ```suggestion
The new `div` element inside `.cat-left-eye` should have the class `cat-left-inner-eye`.
```

```js
assert(document.querySelectorAll('.cat-left-eye div')[0]?.classList.contains('cat-left-inner-eye'));
```

> Your `div` element should have the class `cat-right-inner-eye`.

Reviewer: ```suggestion
The new `div` element inside `.cat-right-eye` should have the class `cat-right-inner-eye`.
```
